### PR TITLE
Fix operator build

### PR DESCRIPTION
**Pull request by dgn**
The current operator build does not work because the binary is called `manager` but the deployment.yaml expects `istio-operator`. Now changed to be  `operator`.

### CODE REVIEW

File: operator/cmd/mesh.go [hunk 1]
```diff
@@ -1,28 +0,0 @@
-// Copyright 2019 Istio Authors
```
Comment (jwendell) [inline operator/cmd/mesh.go]: Is this still in use? There's a whole `mesh` subdir that could be removed as well if this is the case.
`git grep mesh.go` returns some references to this file. Better to remove this and all those references in a separate PR?

File: operator/Makefile.core.mk [hunk 2]
```diff
@@ -1,148 +0,0 @@
-# Copyright 2019 Istio Authors
```
Comment (jwendell) [inline operator/Makefile.core.mk]: Is it safe to remove this file at this point?

**Comment by howardjohn** [comment]
I think makefile removal is good. Not familiar with the rest

File: operator/cmd/mesh.go (hunk 1 above)
Comment (dgn) [reply operator/cmd/mesh.go]: Yeah I added it back for now, it's being used for e2e tests somewhere. Will remove in a future PR

**Review by jwendell — APPROVED** [review]
You need to run make gen and commit the changed files as well...

**Comment by dgn** [comment]
> You need to run make gen and commit the changed files as well...

I'm trying but I keep getting
```invalid configuration: [unable to read client-cert /home/dgrimm/.minikube/client.crt for minikube due to open /home/dgrimm/.minikube/client.crt: no such file or directory, unable to read client-key /home/dgrimm/.minikube/client.key for minikube due to open /home/dgrimm/.minikube/client.key: no such file or directory, unable to read certificate-authority /home/dgrimm/.minikube/ca.crt for minikube due to open /home/dgrimm/.minikube/ca.crt: no such file or directory]```

**Comment by dgn** [comment]
> > You need to run make gen and commit the changed files as well...
> 
> I'm trying but I keep getting
> 
> ```
> invalid configuration: [unable to read client-cert /home/dgrimm/.minikube/client.crt for minikube due to open /home/dgrimm/.minikube/client.crt: no such file or directory, unable to read client-key /home/dgrimm/.minikube/client.key for minikube due to open /home/dgrimm/.minikube/client.key: no such file or directory, unable to read certificate-authority /home/dgrimm/.minikube/ca.crt for minikube due to open /home/dgrimm/.minikube/ca.crt: no such file or directory]
> ```

Weird. It only works when I execute the individual `make` targets, not if I run `make gen`.

File: operator/Makefile.core.mk (hunk 2 above)
Comment (ostromart) [reply operator/Makefile.core.mk]: i don't think all of these have been ported to the root makefile yet?

Comment (dgn) [reply operator/Makefile.core.mk]: I think they have. However, for the sake for easier review (and as this PR requires quite a bit of approvals, apparently), I'm moving the deletion to a separate PR. It's not required to fix the build, anyway.

**Review by howardjohn — APPROVED** [review]

**Pull request merged**